lms/services/exceptions.py

```diff
@@ -173,3 +174,30 @@ class LTIOutcomesAPIError(ExternalRequestError):
     Raised whenever an LTI outcomes API request times out or when an
     unsuccessful, invalid or unexpected response is received from the API.
     """
+
+
+def _exception_class(response):
```

Comment:
This function looks to be entirely Canvas-specific to me, but the containing module is not. Perhaps rename this accordingly (eg. `_canvas_exception_class`)?

Reply:
I've made it a method of the `CanvasAPIError` class that calls it